package-lock.json

```diff
@@ -25355,12 +25355,6 @@
         }
       }
     },
-    "eslint-import-resolver-alias": {
-      "version": "1.1.2",
-      "resolved": "https://registry.npmjs.org/eslint-import-resolver-alias/-/eslint-import-resolver-alias-1.1.2.tgz",
-      "integrity": "sha512-WdviM1Eu834zsfjHtcGHtGfcu+F30Od3V7I9Fi57uhBEwPkjDcii7/yW8jAT+gOhn4P/vOxxNAXbFAKsrrc15w==",
-      "dev": true
-    },
```

Comment:
@tofumatt can the eslint error be caused by this change? Could you please revert it since this doesn't seem to be a relevant change for this PR?

Reply:
> Changing that will cause linting fails so let's ignore that failure for now? It's just the size check anyhow…

I am afraid that if we just ignore it for now, everyone else will have this problem in their PRs which is not what we want. :smile:

Reply:
Oh shoot, good eye. Thanks!

Reply:
That wasn't it, actually 😞 

I'm a bit baffled, I'll see if I can fix that error without causing other ones...

Reply:
I don't think there's any error here: https://github.com/google/site-kit-wp/blob/cbd3f165daf78be82940d9c6456c25ad92876de2/assets/js/components/wp-dashboard/WPDashboardWidgets.js#L78

That's the line it's complaining about. I'm unclear on what's wrong—any idea @aaemnnosttv?